ddp-workspace/projects/ddp-dsm-ui/src/styles/fonts.scss

```diff
@@ -21,3 +21,15 @@ $fonts_directory: '../assets/fonts/Montserrat-';
   font-family: Montserrat-Bold;
   src: url($fonts_directory + 'Bold.ttf');
 }
+
+@mixin MontserratFontFamily($font_name: "regular") {
```

Comment:
Seems like it is never used.

Reply:
Yes, I used it for a while but then I just left it so. 
For sure, we will use it. 
SCSS won't call this code as it's mixin function, therefore don't think this block of code will ever be included in the bundle. 
I mean, we can live it for later uses. 
Please, correct me if I'm wrong

Reply:
Btw, it's just a solution for getting font-family in an easier way, that's why I think we will use it anywhere we need those fonts.

Reply:
It is definitely up to you.
I just like to avoid unused code as other people not to guess why it was added (if they face it in the future).

Reply:
I see. 
There is some chance I can use it even in this PR after integrating the updated API. 
If I didn't use it, I can remove it, though. NP.